content/faq/introduction.md

```diff
@@ -0,0 +1,45 @@
+# Introduction
+
+## Virtual Machine
+The following relate to [running a Vagrant Virtual Machine](../install/vm.md).
```

Comment:
This reorganization of this page is either incomplete or doesn't make sense to me. All the points that are left belong to the troubleshooting section, there is no introduction...

Reply:
What I want to achieve is the separation "you fucked up" vs "we fucked up" but I agree about it being incomplete at the moment. E.g.

- `Port 8000 is already in use` is a user fuck up because he has something on his machine using this port
- `Not receiving beacons` on the contrary is our fuck up

Reply:
Maybe renaming "Introduction" -> "Troubleshooting Vagrant" and "Troubleshooting" -> "Troubleshooting SCION" ?

Reply:
Sorry, I don't see how splitting this improves anything.

Reply:
So you want to have one "FAQ" page containing content of both of them, is that right?

Reply:
That's how I would do it, yes.